yaptide/admin/simulators.py

```diff
@@ -156,7 +156,7 @@ def download_fluka(**kwargs):
     if installation_status:
         click.echo('Fluka installed')
     else:
-        click.echo('Fluka installation failed')
+        click.echo('Not implemented')
```

Comment:
why not implemented ? if installation_status is False then it mean that Fluka download failed. Maybe we should rename `installation_status` to `download_successful`

Reply:
@hendzeld take a look

Reply:
Right, I will change this

Reply:
Updated